controllers/clusterconfig_controller.go

```diff
@@ -59,7 +59,10 @@ type ClusterConfigReconciler struct {
 	BaseURL string
 }
 
-const detachedAnnotation = "baremetalhost.metal3.io/detached"
+const (
+	detachedAnnotation  = "baremetalhost.metal3.io/detached"
+	relocationNamespace = "cluster-relocation"
```

Comment:
Why do we need this?
Can't we keep the resources with the same namespace as they have on the HUB?

Reply:
This would mean every relocated cluster from the same hub would have a different namespace.

Also the secrets can be in a different namespace from the cluster relocation resource so it seemed easier to change everything to one rather than keep track of several namespaces that I need to create.

Reply:
Is there any downside to creating a single namespace in the relocated cluster?

Reply:
No downside, just wonered why not keeping it simple, seems like good enough reasons